web/src/components/snapshots/Snapshots.jsx

```diff
@@ -588,18 +597,23 @@ class Snapshots extends Component {
                   </div>
                 </div>
                 <p className="u-marginTop--10 u-fontSize--normal u-lineHeight--more u-fontWeight--medium u-textColor--bodyCopy">
-                  {" "}
-                  Full snapshots (Instance) back up the Admin Console and all
-                  application data. They can be used for full Disaster Recovery;
-                  by restoring over top of this instance, or into a new cluster.
-                  <span
-                    className="link"
-                    onClick={this.toggleSnaphotDifferencesModal}
-                  >
-                    {" "}
-                    Learn more
-                  </span>
-                  .
+                  {this.props.isEmbeddedCluster
+                    ? "Backups"
+                    : "Full snapshots (Instance)"}{" "}
+                  back up the Admin Console and all application data. They can
+                  be used for full Disaster Recovery; by restoring over top of
+                  this instance, or into a new cluster.{" "}
```

Comment:
Not sure this will display correctly, but I tried.
```suggestion
                  Back up the admin console and all application data
                  for disaster recovery.{" "}
```

Reply:
Looks great!